iceoryx_utils/include/iceoryx_utils/internal/cxx/variant_queue.inl

```diff
@@ -265,6 +298,19 @@ inline bool VariantQueue<ValueType, Capacity>::setCapacity(const uint64_t newCap
             .template get_at_index<static_cast<uint64_t>(VariantQueueTypes::FiFo_MultiProducerSingleConsumer)>()
             ->setCapacity(newCapacity);
     }
+    case VariantQueueTypes::BlockingFiFo_SingleProducerSingleConsumer:
+    {
+        /// @todo must be implemented for FiFo
+        assert(false);
```

Comment:
Could you use Expects or Ensures here? Or maybe even the errorHandler?

Reply:
No the error handler should never be used in the utils! And I see we are using the error handler in this queue - damn.

Reply:
Jup, you're right. This is something for #590. What about Expects or Ensures? I feel un-comfy with plain asserts.

Reply:
Already added